src/utils/useLocationUpdateEffect.ts

```diff
@@ -100,7 +100,7 @@ export const trackVenueEntered = ({
 
 export const leaveRoom = (user: UserInfo) => {
   updateUserProfile(user.uid, {
-    lastSeenAt: 0,
+    lastSeenAt: getCurrentTimeInUnixEpochSeconds(),
```

Comment:
Should this be in seconds, or milliseconds?

(My preference is that we eventually standardise all of these timestamps to use milliseconds, as that is the default in JavaScript)